vmdb/app/controllers/api_controller/vms.rb

```diff
@@ -92,5 +106,16 @@ def destroy_vm(vm)
     rescue => err
       action_result(false, err.to_s)
     end
+
+    def set_owner_vm(vm, owner)
+      desc = "#{vm_ident(vm)} setting owner to '#{owner}'"
+      user = User.find_or_create_by_ldap_upn(owner)
```

Comment:
@Fryguy This is about VM's owner and group. Were you mixing it with User's groups and current group?